lib/flutter_crashlytics.dart

```diff
@@ -2,18 +2,19 @@ import 'dart:async';
 
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
+import 'package:stack_trace/stack_trace.dart';
 
 class FlutterCrashlytics {
   static const MethodChannel _channel =
-      const MethodChannel('flutter_crashlytics');
+  const MethodChannel('flutter_crashlytics');
   static final FlutterCrashlytics _singleton = FlutterCrashlytics._internal();
 
   factory FlutterCrashlytics() => _singleton;
 
   FlutterCrashlytics._internal();
 
   final regexp = RegExp(
-    '([a-zA-Z ]+)(?:\\.(.*))?\\([a-zA-Z:/_]+\\/([0-9a-zA-Z_-]+.dart):([0-9]+):',
+    '([a-zA-Z ]+)(?:\\.(.*))?\\([a-zA-Z:/_]+\\/([0-9a-zA-Z_-]+.dart):([0-9]+):?',
```

Comment:
Can you migrate this one with the library you introduce ? I'll do the iOS part once all the comments has been checked

Reply:
Sure, I will

Reply:
Yeah no problem, for me I prefer merging something coherent than having 2 different way of making stack trace. Please try to keep the same format, at least I'll not have to touch the iOS side for this :)